packages/client/components/GenericAuthentication.tsx

```diff
@@ -99,18 +99,24 @@ const GenericAuthentication = (props: Props) => {
           {counterAction}
         </BrandedLink>
       </DialogSubTitle>
-      <GoogleOAuthButtonBlock isCreate={isCreate} invitationToken={invitationToken} />
-      <HorizontalSeparator margin='1rem 0 0' text='or' />
-      <EmailPasswordAuthForm
-        email={email || ''}
-        isSignin={!isCreate}
-        invitationToken={invitationToken}
-        ref={emailRef}
-        goToPage={goToPage}
-      />
+      {process.env.AUTH_GOOGLE_ENABLED && <GoogleOAuthButtonBlock isCreate={isCreate} invitationToken={invitationToken} />}
```

Comment:
-1 after the vars are moved into createSSR these will probably look like `__AUTH_GOOGLE_ENABLED__`.
You can update the __ACTION__ global var typings here https://github.com/ParabolInc/parabol/blob/eb295331feba7e2534c48012d252c2a52b802905/packages/client/types/modules.d.ts